odpf/assets/v1beta2/experiment.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package odpf.assets.v1beta2;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/odpf/proton/assets/v1beta2;assetsv1beta2";
+option java_outer_classname = "ExperimentProto";
+option java_package = "io.odpf.assets";
+
+// An experiment is the set of configurations and filters that allow for
+// systematically varying some independent variables to impact some other
+// dependent variables.
+message Experiment {
+  // Instance of configurations to be compared in the experiment.
+  message Variant {
+    // Name of the experiment variant.
+    string name = 1;
+
+    // Traffic percent enabled for the variant.
+    float traffic_percent = 2;
+
+    // Indicated whether the variant is the control for the experiment.
+    bool is_control = 3;
+
+    // List of properties the entity has.
+    google.protobuf.Struct attributes = 4;
+
+    // Whether the variant has been promoted to all users.
+    bool is_promoted = 5;
+  }
+
+  // Optional: Type of the entity being experimented over. ex: customer, session,
+  // device, driver etc.
+  string entity = 1;
+
+  // Optional: Percentage of the traffic that the experiment is enabled for.
+  float experiment_traffic = 2;
```

Comment:
@sudo-suhas shall we remove the experiment from here? since it is implied.

Reply:
Also lets make it consistent if we use percentage as keyword here and in variant traiffc.